diff --git a/vendor/magento/module-sales/Model/Order/Pdf/AbstractPdf.php b/vendor/magento/module-sales/Model/Order/Pdf/AbstractPdf.php
index c67024011..b53f6de97 100644
--- a/vendor/magento/module-sales/Model/Order/Pdf/AbstractPdf.php
+++ b/vendor/magento/module-sales/Model/Order/Pdf/AbstractPdf.php
@@ -336,8 +336,9 @@ abstract class AbstractPdf extends \Magento\Framework\DataObject
         $page->setFillColor(new \Zend_Pdf_Color_GrayScale(0));
         $font = $this->_setFontRegular($page, 10);
         $page->setLineWidth(0);
-        $this->y = $this->y ?: 815;
+        $this->y = $this->y ?: 715;
         $top = 815;
+
         $configAddress = $this->_scopeConfig->getValue(
             'sales/identity/address',
             ScopeInterface::SCOPE_STORE,
@@ -358,6 +359,10 @@ abstract class AbstractPdf extends \Magento\Framework\DataObject
                 }
             }
         }
+        $page->drawText(__('Telephone : 01895 55 33 00'), $this->getAlignRight("Telephone : 01895 55 33 00", 130, 440, $font, 10), $top - 12, 'UTF-8');
+		$page->drawText(__('Email : orders@energylightbulbs.co.uk'), $this->getAlignRight("Email : orders@energylightbulbs.co.uk", 130, 440, $font, 10), $top - 24, 'UTF-8');
+		$page->drawText(__('VAT : 987650757'), $this->getAlignRight("VAT : 987650757", 130, 440, $font, 10), $top - 36, 'UTF-8');
+        $top -= 50;
         $this->y = $this->y > $top ? $top : $this->y;
     }
 
@@ -494,10 +499,10 @@ abstract class AbstractPdf extends \Magento\Framework\DataObject
 
         $page->setFillColor(new \Zend_Pdf_Color_GrayScale(0));
         $this->_setFontBold($page, 12);
-        $page->drawText(__('Sold to:'), 35, $top - 15, 'UTF-8');
+        $page->drawText(__('Billing Address:'), 35, $top - 15, 'UTF-8');
 
         if (!$order->getIsVirtual()) {
-            $page->drawText(__('Ship to:'), 285, $top - 15, 'UTF-8');
+            $page->drawText(__('Shipping Address:'), 285, $top - 15, 'UTF-8');
         } else {
             $page->drawText(__('Payment Method:'), 285, $top - 15, 'UTF-8');
         }
@@ -725,26 +730,75 @@ abstract class AbstractPdf extends \Magento\Framework\DataObject
         $lineBlock = ['lines' => [], 'height' => 15];
         foreach ($totals as $total) {
             $total->setOrder($order)->setSource($source);
-
             if ($total->canDisplay()) {
                 $total->setFontSize(10);
                 foreach ($total->getTotalsForDisplay() as $totalData) {
-                    $lineBlock['lines'][] = [
-                        [
-                            'text' => $totalData['label'],
-                            'feed' => 475,
-                            'align' => 'right',
-                            'font_size' => $totalData['font_size'],
-                            'font' => 'bold',
-                        ],
-                        [
-                            'text' => $totalData['amount'],
-                            'feed' => 565,
-                            'align' => 'right',
-                            'font_size' => $totalData['font_size'],
-                            'font' => 'bold'
-                        ],
-                    ];
+                    if($totalData['label'] == "Tax:") {
+                        $lineBlock['lines'][] = [
+                            [
+                                'text' => "20% VAT:",
+                                'feed' => 475,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold',
+                            ],
+                            [
+                                'text' => $totalData['amount'],
+                                'feed' => 565,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold'
+                            ],
+                            [
+                                'text' => "Thank you for your order please call again",
+                                'feed' => 215,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold',
+                            ],
+                        ];
+                    }else if($totalData['label'] == "Subtotal:") {
+                        $lineBlock['lines'][] = [
+                            [
+                                'text' => $totalData['label'],
+                                'feed' => 475,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold',
+                            ],
+                            [
+                                'text' => $totalData['amount'],
+                                'feed' => 565,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold'
+                            ],
+                            [
+                                'text' => "Damaged good must be reported within 5 days of delivery",
+                                'feed' => 280,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold',
+                            ],
+                        ];
+                    }else{
+                        $lineBlock['lines'][] = [
+                            [
+                                'text' => $totalData['label'],
+                                'feed' => 475,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold',
+                            ],
+                            [
+                                'text' => $totalData['amount'],
+                                'feed' => 565,
+                                'align' => 'right',
+                                'font_size' => $totalData['font_size'],
+                                'font' => 'bold'
+                            ],
+                        ];
+                    }
                 }
             }
         }
diff --git a/vendor/magento/module-sales/Model/Order/Pdf/Invoice.php b/vendor/magento/module-sales/Model/Order/Pdf/Invoice.php
index d4ce16d1b..73170ae58 100644
--- a/vendor/magento/module-sales/Model/Order/Pdf/Invoice.php
+++ b/vendor/magento/module-sales/Model/Order/Pdf/Invoice.php
@@ -92,15 +92,15 @@ class Invoice extends AbstractPdf
         //columns headers
         $lines[0][] = ['text' => __('Products'), 'feed' => 35];
 
-        $lines[0][] = ['text' => __('SKU'), 'feed' => 290, 'align' => 'right'];
+        $lines[0][] = ['text' => __('SKU'), 'feed' => 330, 'align' => 'right'];
 
         $lines[0][] = ['text' => __('Qty'), 'feed' => 435, 'align' => 'right'];
 
-        $lines[0][] = ['text' => __('Price'), 'feed' => 360, 'align' => 'right'];
+        $lines[0][] = ['text' => __('Price'), 'feed' => 395, 'align' => 'right'];
 
-        $lines[0][] = ['text' => __('Tax'), 'feed' => 495, 'align' => 'right'];
+        $lines[0][] = ['text' => __('VAT'), 'feed' => 495, 'align' => 'right'];
 
-        $lines[0][] = ['text' => __('Subtotal'), 'feed' => 565, 'align' => 'right'];
+        $lines[0][] = ['text' => __('Total'), 'feed' => 565, 'align' => 'right'];
 
         $lineBlock = ['lines' => $lines, 'height' => 5];
 
@@ -123,7 +123,7 @@ class Invoice extends AbstractPdf
         $pdf = new \Zend_Pdf();
         $this->_setPdf($pdf);
         $style = new \Zend_Pdf_Style();
-        $this->_setFontBold($style, 10);
+        $this->_setFontBold($style, 8);
 
         foreach ($invoices as $invoice) {
             if ($invoice->getStoreId()) {
@@ -160,6 +160,7 @@ class Invoice extends AbstractPdf
                     continue;
                 }
                 /* Draw item */
+               // $page->setLineWidth(0.5);
                 $this->_drawItem($item, $page, $order);
                 $page = end($pdf->pages);
             }
